src/screens/Asl/components/AslTable/index.js

```diff
@@ -147,16 +147,22 @@ const AslTable = props => {
     return (
       <div>
         {showVideo && (
-          <div class="video-window">
+          <div className="video-window">
             <button 
               id='close-button'
-              onClick={() => setShowVideo(false)}>
+              onClick={() => setShowVideo(false)}
+            >
               X
             </button>
-            <video  class="video-js vjs-default-skin" controls
-              preload="auto" width="640" height="264"
-              data-setup="{}">
-              <source src={videoUrl} type='video/mp4'></source>
+            <video 
+              className="video-js vjs-default-skin" 
+              controls
+              preload="auto" 
+              width="640" 
```

Comment:
Is there a better way to do width and heigh that is more dynamic and flexible e.g. works with displays of very size?

Reply:
I have try to use the original size of the video, but it turns out to be very large. I can try to resize the window and keep the width and height ratio.

Reply:
My concern is "640" can be a very different size e.g. on Mobile e.g. on wide screen e.g. on laptop screen.
What's the best practice for adaptive layouts? A % width of the screen??

Reply:
I see. I will fix that.

Reply:
Fixed with new commit and style